packages/react-components/react-color-picker-preview/stories/src/ColorPicker/ColorPickerDefault.stories.tsx

```diff
@@ -107,3 +177,17 @@ const handleOnBlur = (e: React.FocusEvent<HTMLInputElement>) => {
     e.target.removeAttribute('aria-invalid');
   }
 };
+
+const handleRgbKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
is there any way to get rid of this handler? I still don't get it why can't we use the controlled input with onChange/onInput only, and don't use prevent default and keys whitelist, etc

Reply:
I'm working on it. When I use restriction in onChange then SpinButton stops working from keyboard. Or if I put preventDefault to SpinButton onChange it also doesn't work